### ISSUE TITLE
stuck at "searching for HEOS devices ..."

### ISSUE THREAD

**Issue by TheMiller2015** [withstu/ioBroker.heos#87]
Hello I try to implement your adapter but after creating an instance, it stucks at "searching for HEOS devices ..." (log). 

I have an denon av receiver, one heos 150 speaker, iobroker installed in a docker-container (but "network_mode: host" because of the multicast for heos) and also loaded the node module node-ssdp within the javascript.0-instance. What else can I do? Can you help me?

Thanks in advance

**Comment by withstu** [comment]
As I don't use docker with this adapter I can only give you some hints, what you can try:
- enable debug log of the adapter and look for ssdp activity
- with the console go to the adapter installation in you container. Then run the following node script. If you don't see any responses, you should again check your network configuration: https://github.com/withstu/ioBroker.heos/blob/master/test/node-ssdp/index.js

**Comment by TheMiller2015** [comment]
Thanks for your fast answer. I tried both. The problem seems to be that hyper-v (which hosts the linux guest) don't support multicast by default. So I'll take a look at this configuration. I'm running a Windows-Server, that hosts a linux guest using hyper-v as hypervisor. This linux-guest has docker installed and runs iobroker and some other containers. Never had a problem with this, but I never needed multicast before. I'll update this issue for users if I made any progress.
But, feel free to help anyway :)
